streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -192,12 +191,11 @@ public String toString() {
      */
     @Override
     public void configure(final Map<String, ?> configs) {
-        final AssignorConfiguration assignorConfiguration = new AssignorConfiguration(configs);
+        assignorConfiguration = new AssignorConfiguration(configs);
```

Comment:
Thanks. Should we instead make `referenceContainer` a field and access its members to get the references? The purpose of `AssignorConfiguration` is to parse the configs, not to be a general container for the configured values. Otherwise, we wouldn't need to assign any of the other fields here.

Reply:
Sure.

Reply:
Thanks for the update! Are you still planning to drop the `assignorConfiguration` field in favor of a `referenceContainer` field?

Reply:
I never intended to drop it. Maybe I miss understand your comment?

We could replace the field `StreamsParttionAssignor#taskManager` etc with `StreamsPartitionAssigner#referenceContainer` but it just make the code lines longer each time we need to access the TM (etc). Thus, it seems to make the code more readable if we just "extract" those field from the reference container once?